OG_Format.adoc

```diff
@@ -133,8 +133,8 @@ Name of the platform(s) that supported the sensors data used to create this data
 
 https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/collection/L06/current/_]
 
- |mandatory |“Autonomous Underwater Vehicle”
-|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
+ |mandatory |“”
```

Comment:
Isn't missing 'sub-surface gliders' ?

Reply:
Yes. 
Error in the change. I will deleate and re-do the PR.

Reply:
I just edited it for you.